locations/spiders/homedepot.py

```diff
@@ -15,7 +17,12 @@ class HomeDepotSpider(SitemapSpider):
     ]
 
     def parse_store(self, response):
-        item = LinkedDataParser.parse(response, "LocalBusiness")
+        json_ld = json.loads(
+            response.xpath(
+                '//script[@id="thd-helmet__script--storeDetailStructuredLocalBusinessData"]/text()'
```

Comment:
I can't access Home Depot website from the UK, but does it not include `type="application/ld+json"`? If it does the normal `item = LinkedDataParser.parse(response, "LocalBusiness")` should work.

Reply:
It does include JSON-LD but as I mentioned in the PR description I couldn't figure out how to specify a custom time format with `LinkedDataParser.parse()`. Sorry, I should have added comments in the code to make that clearer.

Reply:
My bad, I didn't read that. Ideally we should fix that.